configure.py

```diff
@@ -1557,7 +1557,7 @@ def find_ninja():
     "test/boost/linearizing_input_stream_test.cc",
     "test/lib/log.cc",
```

Comment:
> In the next patch we want to make schema/schema.o depened on
alternator/executor.o because when the schema has an Alternator
computed column, the schema code needs to construct the computed column
object (extract_from_attrs_column_computation) and that lives in
alternator/executor.o.

It's better to promote the extract_from_attrs_column_computation to scylla core, or abstract it behind an interface. Otherwise, this is another blocker for the modularization effort, if everything depends on everything else.

Reply:
@avikivity, if you agree I want to get this feature in and later do this rewrite. I'll open an issue about it.

Reply:
ok